roles/unbind-keycloak-apb/tasks/main.yml

```diff
@@ -20,13 +18,14 @@
     var: keycloak_auth_response
     verbosity: 2
 
--
-  name: delete client {{ _apb_bind_creds.resource }} in realm {{ namespace }}
+- set_fact: KEYCLOAK_REALM={{ _apb_bind_creds.bearer_installation.realm}}
+
+- name: delete client {{ _apb_bind_creds.resource }} in realm {{ KEYCLOAK_REALM }}
```

Comment:
Why client is named `_apb_bind_creds.resource` and not `_apb_bind_creds.client`?

Reply:
This is again from the broker. Essentially it passes back what you created during the bind operation

Reply:
The resource will have the actual client name thats displayed in the realm. the client_id is used in the url.